README.md

```diff
@@ -43,13 +44,12 @@ Set up a vhost that points to the `/public` folder:
 
 ## Inserting new data
 
-Use `insert.php` to insert json files. This file uses the `database.php` config file in `application/config` so be sure it's set up correctly.
- 
-The first argument of the script is the path to the file you want to insert. The second argument is the version. Example:
 
-```
-php insert.php application/files/reports_2019-06-18_develop.json develop
-```
+Use the hook provided in the `Hook` controller. You need to call this URL: `BASE_URL/hook/add` with the following GET parameters:
+- `token`: the token set in the environment variable `QANB_TOKEN` (e.g.: `IpBzOmwXQUrW5Hn`)
+- `filename` : the complete filename to look for in the Google Cloud Storage (e.g.: `2019-07-22-develop.json`). The name must follow this pattern: `/[0-9]{4}-[0-9]{2}-[0-9]{2}-(.*)?\.json/`
+Optional:
+- `force`: a special parameter used to force insert when a similar entry is found (criterias are date and version)
```

Comment:
Also indicate that the files will be taken from https://storage.googleapis.com/prestashop-core-nightly/reports/ (so it will not work if given filename is in https://storage.googleapis.com/prestashop-core-nightly/anotherfolder/)

Reply:
Done thanks